dist/mushroom-strategy.js

```diff
@@ -1,1335 +1,1205 @@
+                "switch.turn_off",
+                "switch.turn_off",
+                "mdi:power-plug-off",
+                "mdi:power-plug",
+                area.area_id,
+            ),
+        ),
+      });
+    }
+
+    // Create a column of climate cards.
+    const climates = this.#getDeviceEntities(area, "climate");
+
+    if (climates.length) {
+      cards.push({
+        type: "vertical-stack",
+        cards: this.#createViewCards(
+            climates,
+            entityConfig,
+            {
+              type: "custom:mushroom-climate-card",
+              hvac_modes: [
+                "off",
+                "cool",
+                "heat",
+                "fan_only",
+              ],
+              show_temperature_control: true,
+            },
+            {
+              type: "custom:mushroom-title-card",
+              subtitle: "Climate",
+            },
+        ),
+      });
+    }
+
+    // Create a column of Media player cards.
+    const mediaPlayers = this.#getDeviceEntities(area, "media_player");
+
+    if (mediaPlayers.length) {
+      cards.push({
+        type: "vertical-stack",
+        cards: this.#createViewCards(
+            mediaPlayers,
+            entityConfig,
+            {
+              type: "custom:mushroom-media-player-card",
+              use_media_info: true,
+              media_controls: [
+                "on_off",
+                "play_pause_stop",
+              ],
+              show_volume_level: true,
+              volume_controls: [
+                "volume_mute",
+                "volume_set",
+                "volume_buttons",
+              ],
+            },
+            {
+              type: "custom:mushroom-title-card",
+              subtitle: "Media Players",
+            },
+        ),
+      });
+    }
+
+    // Create a column of Sensor cards.
+    let sensors        = this.#getDeviceEntities(area, "sensor");
+    const sensorStates = this.#getStateEntities(area, "sensor");
+    const sensorCards  = [];
+
+    if (sensors.length) {
+      // Add a Title card.
+      sensorCards.push({
+        type: "custom:mushroom-title-card",
+        subtitle: "Sensors",
+      });
+
+      // Create a card for each sensor.
+      for (const sensor of sensors) {
+        // Find the state of the current sensor.
+        const sensorState = sensorStates.find(state => state.entity_id === sensor.entity_id);
+
+        // Define the card type.
+        let card = {
+          type: "custom:mushroom-entity-card",
+          entity: sensor.entity_id,
+          icon_color: "green",
+        };
+
+        if (sensorState.attributes.unit_of_measurement) {
+          card = {
+            type: "custom:mini-graph-card",
+            entities: [sensor.entity_id],
+            animate: true,
+            line_color: "green",
+          };
+        }
+
+        if (entityConfig) {
+          card = entityConfig.find(config => config.entity_id === sensor.entity_id);
```

Comment:
There is no check if the returned value is null. Because of this, when I have an `entity_config`, all of the sensor cards are null.

Reply:
I need some more clarification.
When would it be null?
Can you give a sample of you configuration?